server/connections.py

```diff
@@ -103,6 +118,16 @@ async def comm_loop(self) -> None:
                         dict(key=event.data["value"],  userid=self.identity)
                     )
                 await self.manager.broadcast_buffer(self.team)
+
+                # if the buffers are now the same length
+                if len(self.manager.buffers[self.team]) == prompt_length:
```

Comment:
Won't this always be false, since it's cleared above?

Reply:
We add the element after the first check